apps/catalogue.code-workspace

```diff
@@ -5,6 +5,12 @@
 		},
 		{
 			"path": "tailwind-components"
+		},
+		{
+			"path": "ui"
+		},
+		{
+			"path": "meta-data-utils"
```

Comment:
is this of general use or vscode specific? I don't mind but we might benefit from some explanation on how to use this then. (I know, this already existed before)